beetsplug/convert.py

```diff
@@ -491,6 +503,26 @@ def convert_func(self, lib, opts, args):
         pipe = util.pipeline.Pipeline([iter(items), convert])
         pipe.run_parallel()
 
+        if opts.prune or self.config['prune'].get():
```

Comment:
Configuration options are automatically converted to `bool`:

```suggestion
        if opts.prune or self.config['prune']:
```